src/applications/mhv-supply-reordering/pages/chooseSupplies.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+// import { validateAtLeastOneSelected } from '../utils/validators';
+import { checkboxGroupSchema } from '@department-of-veterans-affairs/platform-forms-system/web-component-patterns';
+
+import {
+  suppliesReplaceSchema,
+  suppliesUpdateUiSchema,
+  suppliesUi,
+} from '../utils/helpers';
+
+const numberOfSuppliesPhrase = count => {
```

Comment:
maybe we could move this to a shared utility and use it other places the pattern occurs? (I know I saw it at least one other place)